### DISCUSSION TITLE
Added features?

### DISCUSSION THREAD

**Discussion by ExTechOp** [piggz/harbour-advanced-camera#131] [Ideas]
First, thanks for putting the work into creating this. Would it be possible to look at some features that I feel could enhance camera usage?
- "digital leveling" gauge to indicate whether or not the camera is level (both the horizontal and front-back directions)
- on multi-camera phones the possibility of simultaneously triggering multiple cameras, to get different framings/zooms
- possibility of taking multiple consecutive shots within a short timeframe, for catching quick things happening
How do people feel about these ideas?

**Comment by Karry** [comment]
I like these ideas, but I am worry that at least "triggering multiple cameras" and "consecutive shots within a short timeframe" would not be easy, or possible. Because advanced camera is using camera QML api and it has no such advanced features as I know.
